### ISSUE TITLE
macOS compatibility

### ISSUE THREAD

**Issue by JavierSanchez-Utges** [tiwarylab/GrASP#6]
Is GrASP compatible with macOS? I think it might not be as certain dependencies, most of them CUDA-related seem not to be compatible with this OS.

**Comment by zwsmith200** [comment]
This is likely related to the [pytorch install](https://pytorch.org/get-started/locally/). We can add a version without CUDA as part of #7 .

**Comment by JavierSanchez-Utges** [comment]
Yes, I believe all the libraries without macOS support are NVIDIA CUDA ones. CUDA-less version sounds great! Thanks!
